baitap-19-10-2020/bai-2.js

```diff
@@ -0,0 +1,10 @@
+const yearOld = (age) => {

```

Comment:
tên hàm phải là động từ
```suggestion
const checkIsHighSchoolAge = age => {

```